examples/upload-files

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+import os
+import logging
+from argparse import ArgumentParser
+
+from pubtools.pulplib import Client
+
+log = logging.getLogger("upload")
+
+
+def upload(client, path, repo_id):
+    repo = client.get_repository(repo_id).result()
+
+    uploads = []
+    if os.path.isdir(path):
+        for file in os.listdir(path):
+            if os.path.isfile(file):
```

Comment:
I just gave this a try locally and I don't think it works (or it may only work in the case where path is ".")

os.listdir returns filenames relative to the directory being listed.  If path isn't the current working directory then those files won't be accessible by the relative paths.

e.g. if path is /some/directory and /some/directory/file1, /some/directory/file2 exist, then "file" would be e.g. "file1", os.path.isfile("file1") will return False unless the current working directory happens to be /some/directory.

Looks like the os.path.join should be moved to before this isfile check and then it'd work.